debian/control

```diff
@@ -1,15 +1,18 @@
 Source: lutris
 Section: games
 Priority: optional
-Build-Depends: cdbs,
- debhelper,
+Build-Depends: debhelper,
+ libsdl2-2.0-0:i386,
+ libstdc++6:i386,
+ libgcc1:i386,
+ libc6:i386,
  python,
  dh-python,
  gir1.2-gtk-3.0,
  gir1.2-glib-2.0,
  python-gi,
- libgirepository1.0-dev,
-Maintainer: Mathieu Comandon <strider@strycore.com>
+ libgirepository1.0-dev
+Maintainer: Michael DeGuzis <mdeguzis@gmail.com>
```

Comment:
Ok, so I'm going to release 0.3.7.3 today (using the older packaging system, the packages are already built) and then remove koku-xinput-wine from the project and move it to the lutris runtime, where it belongs. This should solve the issue. Keep an eye for activity on the Next branch in the next few hours.